capi-eks-multi-tenancy/eks_related_yaml_template.sh

```diff
@@ -0,0 +1,95 @@
+function generate_static_provider_cr() {
+cat <<EOF > capi-eks-static.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cluster.x-k8s.io/ns: "testlabel"
+  name: test-eks
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: AWSClusterStaticIdentity
+metadata:
+  name: "test-account"
+spec:
+  secretRef: "test-account-creds"
```

Comment:
it only need secret name, and it doesn't support namespace in v1alpha4, so that can still be a limitation for us. If using static identity (key+keyid) users have to create IAMs in the same namespace (capa controller namespace)